core/src/main/scala/stryker4s/config/Config.scala

```diff
@@ -11,8 +11,9 @@ case class Config(mutate: Seq[String] = Seq("**/main/scala/**/*.scala"),
                   reporters: List[MutantRunReporter] = List(new ConsoleReporter),
                   logLevel: Level = Level.INFO,
                   files: Option[Seq[String]] = None,
-                  excludedMutations: Set[String] = Set.empty,
-                  thresholds: Thresholds = Thresholds()) {
+                  excludedMutations: ExcludedMutations = ExcludedMutations(),
+                  thresholds: Thresholds = Thresholds(),
+                  systemProperties: Set[String] = Set.empty) {
```

Comment:
Shouldn't this be in a case class as well (or a `Set[SomeOtherType]`) to prevent problems should we ever want to add another set?

Reply:
Having another `Set` is not an issue in itself. I had to change it because ExcludedMutations has special behaviour: it only allows predefined values (EqualityOperator, BooleanLiteral, LogicalOperator, etc). 

`systemProperties` is just a normal `Set` so I don't think it needs a case class. :)